python/ray/serve/_private/controller.py

```diff
@@ -740,13 +740,17 @@ def deploy_application(self, name: str, deployment_args_list: List[bytes]) -> No
                     "deployment_config_proto_bytes": deployment_args.deployment_config,
                     "replica_config_proto_bytes": deployment_args.replica_config,
                     "deployer_job_id": deployment_args.deployer_job_id,
-                    "route_prefix": deployment_args.route_prefix
-                    if deployment_args.HasField("route_prefix")
-                    else None,
+                    "route_prefix": (
+                        deployment_args.route_prefix
+                        if deployment_args.HasField("route_prefix")
+                        else None
+                    ),
                     "ingress": deployment_args.ingress,
-                    "docs_path": deployment_args.docs_path
-                    if deployment_args.HasField("docs_path")
-                    else None,
+                    "docs_path": (
+                        deployment_args.docs_path
+                        if deployment_args.HasField("docs_path")
+                        else None
+                    ),
```

Comment:
Linter seemed to want these changes, not sure why they weren't already like this 🤔